communication/best-practices.md

```diff
@@ -0,0 +1,85 @@
+[Special Interest Groups (SIG)][sig-list] and [Working Groups (WG)][sig-list].  
+That's why you may want to set your filters in your Gmail account to attain a
+good signal-to-noise ratio with regards to the mailing list messages and Github
+notifications.
+
+- [Example for setting filters to Kubernets Mailing lists](#example-for-setting-filters-to-kubernets-mailing-lists)
+- [Examples for setting filters to Kubernetes Github notifications](#examples-for-setting-filters-to-kubernetes-github-notifications)
+                                                                                
+## Example for setting filters to Kubernets Mailing lists           
+
+It depends on the SIG or/and WG you are involved in, you would like to setup
+your filters in your Gmail account, so you don't need to have all emails
+structred in one place.
+Create a filter following the procedure below:
+
+- In your Gmail account click on **Settings**:
+  - **Filters and Blocked Addresses** -> Scroll down and click **create a new filter**
+  - In the **to** fields write the email of the SIG's Google Group. 
+  - **Create filter** -> Check the box ** Apply the label** and create new
+label by choosing **New label...** in the dropdown list.
+  - Click on the **Create filter**.
+
+## Examples for setting filters to Kubernetes Github notifications
+
+In order to not get spammed by Github notifications and to get all the 
+notifications in the right place, here is an example of filters to create/apply
+on your Gmail account:
+
+- Stick a blue label on anything kubernetes-related:
+  - Matches: (kubernetes OR k8s OR kubelet OR kube
+  - Do this: Apply label "k8s", Mark it as important
+- Archive my own actions (sending these is an option in github's settings).  
+I send them but archive them, so if I need to see history of an issue I can:
+  - Matches: from:noti...@github.com to:(your_a...@noreply.github.com) 
+    - Do this: Skip Inbox, Mark as read
+- Skip bot comments:
+  - Matches: (from:(notifi...@github.com) (from:(k8s-merge-robot) OR 
+from:(Kubernetes Bot) OR from:(k8s-ci-robot) OR from:(Submit Queue)))
+    - Do this: Skip Inbox, Mark as read
+- Skip push notifications:
+  - Matches: to:(pu...@noreply.github.com) 
+    - Do this: Skip Inbox, Mark as read 
+- Red label on things assigned to me:
+  - Matches: to:(ass...@noreply.github.com)
+    - Do this: Star it, Apply label "gh/p0", Mark it as important 
+- Orange label:
+  - Matches: to:(com...@noreply.github.com) 
+    - Do this: Star it, Apply label "gh/p1" 
+- Yellow label:
+  - Matches: to:(men...@noreply.github.com) 
+    - Do this: Apply label "gh/p2" 
+- Grey label:
+  - Matches: to:(team_m...@noreply.github.com)
+    - Do this: Apply label "gh/p3"
+  - Matches: to:(aut...@noreply.github.com) 
+    - Do this: Star it, Apply label "gh/me", Mark it as important 
+- Red label:
+  - Matches: (list:"kubernetes.kubernetes.github.com" ("Labelling this PR 
```

Comment:
Maybe add a note that you can also categorize per repo?

For example, I use this:

```
Matches: list:(community.kubernetes.github.com)
Do this: Apply label "k8s/community
```